### ISSUE TITLE
bug: aria-disabled is not automatically applied

### ISSUE THREAD

**Issue by kensodemann** [ionic-team/ionic-framework#23513]
# Bug Report

**Ionic version:**

[ ] **4.x**
[x] **5.x**
[ ] **6.x**

**Current behavior:**
If you add `disabled={true}` to an `IonRadio` or `IonToggle` (and perhaps others, but these are the ones I tested) `aria-disabled` is not automatically applied. As a result, AXE flags these for a color contrast violation when they should be ignored.

Playing around with this a bit, it would be best to also apply `aria-disabled` to the label associated with the toggle or radio.

In my experiment, disabling the `IonSelect` resulted in adding `aria-disabled` to the select, but not the associated label, resulting in the label being flagged.

**Expected behavior:**
Ideally, automatically add `aria-disabled` to the components that are visually affected by the `disabled={true}` on the interactive component.

**Steps to reproduce:**
Use this repro case: https://codesandbox.io/s/ionic-axe-42-failures-1y1pq?file=/src/App.tsx

Add `disabled={true}` to the `IonSelect`, `IonToggle`, and `IonSelect` and examine. Run the AXE devtools. Add `aria-disabled` to the components that need it, and repeat.
